flax/linen/recurrent.py

```diff
@@ -392,6 +392,113 @@ def num_feature_axes(self) -> int:
     return 1
 
 
+class SimpleCell(RNNCellBase):
+  r"""Simple cell.
+
+  The mathematical definition of the cell is as follows
+
+  .. math::
+
+      \begin{array}{ll}
+      h' = \tanh(W_i x + b + W_h h)
```

Comment:
Consider using `b_i `to denote which dense layer the bias is associated with, since it will be reflected in the params tree structure returned with `.init`